std/http.ln

```diff
@@ -46,6 +35,109 @@ export type Response {
   connId: int64,
 }
 
+/**
+ * HTTP Client
+ */
```

Comment:
The new client code became a *lot* bigger than I had hoped for since I couldn't implement `Result`'s `map` function. Really want function types! :)